pkgs/tools/networking/mu/default.nix

```diff
@@ -50,6 +50,6 @@ stdenv.mkDerivation rec {
     homepage = "https://www.djcbsoftware.nl/code/mu/";
     changelog = "https://github.com/djcb/mu/releases/tag/v${version}";
     maintainers = with maintainers; [ antono chvp peterhoeg ];
-    platforms = platforms.mesaPlatforms;
+    platforms = [ "i686-linux" "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
I don't think it's very common to list specific platforms here. The `platforms.unix` list makes more sense here I think?

Reply:
Makes perfect sense. 

The rest of the noise in that file is just due to `nixpkgs-fmt`.